lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -75,20 +90,38 @@ module.exports = function (
       return $q.resolve(false)
     }
 
-    if (constraints.isDriveSizeRecommended(drive, selectionState.getImage())) {
-      return $q.resolve(true)
+    const isLargeDrive = constraints.isDriveSizeLarge(drive)
+    const isRecommendedDrive = constraints.isDriveSizeRecommended(drive, selectionState.getImage())
```

Comment:
Minor nit, but could we rename this variable to `isRecommendedSizeDrive` ?
To me, a variable named `isRecommendedDrive` implies that this drive is recommended above all other drives :)